src/app/family-deliveries/family-deliveries.component.ts

```diff
@@ -662,6 +663,35 @@ export class FamilyDeliveriesComponent implements OnInit, OnDestroy {
     };
   }
 
+  
+  @ServerFunction({ allowed: Roles.lab })
+  static async getDeliveriesByPhone(phoneNum: string, context?: Context, db?: SqlDatabase) {
+    let sql1 = new SqlBuilder();
+    let fd = context.for(FamilyDeliveries).create();
+    let result : string[] = [];
+    let courier = await (await context.for(Helpers).findFirst(i => i.phone.isEqualTo(phoneNum)));
+
+    for (const d of (await db.execute(sql1.query({
+      from: fd,
+      where: () => [
+        (courier != undefined ? fd.courier.isEqualTo(courier.id).and(fd.active()) :
+        sql1.or(
+          fd.phone1.isEqualTo(phoneNum).and(fd.active()),
```

Comment:
recommend to fix tis filter in seveal ways:

- [ ] do active and (x or y or z)
- [ ] support the case when some one can be both a volunteer and a donor - (item one will fix that)
- [ ] improve the phone search - use what I've done in `checkDuplicateFamilies`, so to ignore hyphens etc...